ang/test/civiawards/award-creation/directives/review-panels/review-panels.directive.spec.js

```diff
@@ -271,58 +266,138 @@
     });
 
     describe('review panels list', () => {
-      beforeEach(() => {
-        $scope.awardId = '10';
-        createController();
-        $scope.$digest();
-      });
+      describe('when there is atleast one relationship', () => {
+        beforeEach(() => {
+          $scope.awardId = '10';
+          createController();
+          $scope.$digest();
+        });
 
-      it('fetches all exisiting review panels for the award', () => {
-        expect(crmApi).toHaveBeenCalledWith('AwardReviewPanel', 'get', {
-          sequential: 1,
-          case_type_id: '10',
-          options: { limit: 0 }
+        it('fetches all exisiting review panels for the award', () => {
+          expect(crmApi).toHaveBeenCalledWith('AwardReviewPanel', 'get', {
+            sequential: 1,
+            case_type_id: '10',
+            options: { limit: 0 }
+          });
+        });
+
+        it('displays all existing review panels for the award', () => {
+          expect($scope.existingReviewPanels).toEqual([{
+            id: '46',
```

Comment:
Can we save this object into a new variable and store it in `mocks/data` folder ?

Reply:
Moved to a new variable, as discussed.